geometry_msgs/Point32[] dig_cloud
geometry_msgs/Point32[] fid_cloud